django/utils/http.py

```diff
@@ -277,12 +277,15 @@ def is_same_domain(host, pattern):
     )
 
 
-def is_safe_url(url, host=None):
+def is_safe_url(url, host=None, secure=False):
     """
     Return ``True`` if the url is a safe redirection (i.e. it doesn't point to
     a different host and uses a safe scheme).
 
     Always returns ``False`` on an empty url.
+
+    If ``secure`` argument is ``True``, only https will be considered as a
+    valid scheme as opposed to both http and https with default ``False``.
```

Comment:
We also need to add a release note for the new argument in `docs/releases/1.11.txt`.